index.d.ts

```diff
@@ -0,0 +1,129 @@
+/** 
+ * Typings for twitter-lite 
+ * 
+ * @version 0.9.4
+ * @author Floris de Bijl <@fdebijl>
+*/
+
+/// <reference types="node" />
+
+import { EventEmitter } from 'events';
+import { Header } from 'oauth-1.0a';
+
+declare namespace TwitterLite {
+  type TwitterOptions = {
```

Comment:
when possible interface is preferable over type
https://www.typescriptlang.org/docs/handbook/advanced-types.html#interfaces-vs-type-aliases